docker/init.sh

```diff
@@ -0,0 +1,3 @@
+bin/rails db:create
+bin/rails db:migrate
+hivemind -p 3000 ./Procfile.docker.dev
```

Comment:
```suggestion
hivemind -p 5000 ./Procfile.docker.dev
```

Reply:
See comment below about exposing ports in `docker-compose.yml`